### DISCUSSION TITLE
Password reveal

### DISCUSSION THREAD

**Discussion by robertjmccarthy** [ministryofjustice/moj-frontend#709] [ARCHIVE: Components, pages and patterns]
The [password reveal component](https://design-patterns.service.justice.gov.uk/components/password-reveal/) is live in the MoJ Design System.

Use this discussion to suggest changes and improvements to this component.

**Comment by helennickols** [comment]
The [password reveal component](https://design-patterns.service.justice.gov.uk/components/password-reveal/) has been archived. 

## If your service contains the password reveal component

You don’t need to do anything immediately because this isn’t a breaking change. 

The component is no longer being maintained so it’ll never get an accessibility update or bug fix. You need to switch all instances of it to the [GOV.UK Design System password input component](https://design-system.service.gov.uk/components/password-input/) to benefit from these types of improvements. 

## Why we archived it

- The [GOV.UK Design System password input component](https://design-system.service.gov.uk/components/password-input/) does the same job – there’s no need for both.
- Archiving it means we don’t need to maintain it anymore. This frees up our time for more important stuff. 🙌🎊
- MoJ teams were supportive of archiving it.
